lib/connectors/openid.js

```diff
@@ -72,11 +77,17 @@ module.exports = function (openIdConfiguration, scimConfiguration = {}) {
             response.locals.authx = introspectionResponse;
             openIdClient.userinfo(token).then((userInfo) => {
               response.locals.authx.uuid = userInfo.inum;
+              if (this.cb) {
```

Comment:
Not sure if this will work as expected. `this` is within the context of a handler function inside the `then` logic. Instead, you could declare a constant in `module.exports` and use that constant here.

Has this been tested?

Reply:
Yeah. this is working.